Content.Client/Stylesheets/IStylesheetManager.cs

```diff
@@ -1,12 +1,32 @@
+using System.Diagnostics.CodeAnalysis;
 using Robust.Client.UserInterface;
 
-namespace Content.Client.Stylesheets
+namespace Content.Client.Stylesheets;
+
+public interface IStylesheetManager
 {
-    public interface IStylesheetManager
-    {
-        Stylesheet SheetNano { get; }
-        Stylesheet SheetSpace { get; }
+    /// Nanotransen styles: the default style! Use this for most UIs
+    Stylesheet SheetNanotransen { get; }
```

Comment:
"Nanotransen" typo - also appears in a handful of other places.

Reply:
I honestly fully believed thats how you spelt Nanotrasen